app/api/v1/manual_tests/recommendations.http

```diff
@@ -0,0 +1,31 @@
+POST http://localhost:5000/api/v1/recommendations?location=services_list&version=v1
+Content-Type: application/json
+
+{
+    "user_id": 1,
+    "recommendation_page_visit_id": 1,
+    "search_phrase": "GPU cluster",
+    "filters": [
+        "filter_name_1",
+        "filter_name_2",
+        "filter_name_3"
+    ]
+}
+
+###
+
+POST http://localhost:5000/api/v1/recommendations?location=services_list&version=v2
+Content-Type: application/json
+
+{
+    "user_id": 1,
+    "recommendation_page_visit_id": 1,
+    "search_phrase": "GPU cluster",
+    "filters": [
+        "filter_name_1",
+        "filter_name_2",
+        "filter_name_3"
+    ]
+}
+
+###
```

Comment:
I'm not entirely sure whether those should be part of the repository. But it might be, if we decide that `*.http` files should be part of it.

Reply:
yes, it was obsolete, I removed it.